os/board/rtl8730e/src/component/mbed/targets/hal/rtl8730e/i2s_api.c

```diff
@@ -216,7 +216,7 @@ void i2s_set_dma_buffer(i2s_t *obj, char *tx_buf, char *rx_buf,
 {
 	assert_param(IS_SP_SEL_I2S(obj->i2s_idx));
 
-	uint32_t i, j;
+	__attribute__((unused)) uint32_t i, j;
```

Comment:
Could you please revert this? I will make necessary changes

Reply:
ok.. I will revert in next commit